<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bio Tool 1</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="ImageView" name="imageView" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actNew"/>
    <addaction name="actOpen"/>
    <addaction name="actSave"/>
    <addaction name="actSaveAs"/>
    <addaction name="actExport"/>
    <addaction name="actExportOption"/>
    <addaction name="separator"/>
    <addaction name="actLoadImage"/>
    <addaction name="actCloseImage"/>
    <addaction name="separator"/>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actResetView"/>
    <addaction name="separator"/>
    <addaction name="actModeView"/>
    <addaction name="actModeEdit"/>
    <addaction name="actModeDefineCentroid"/>
    <addaction name="separator"/>
    <addaction name="actSettings"/>
   </widget>
   <widget class="QMenu" name="menu_Aide">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actAbout"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menu_Aide"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Barre d'outils</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actNew"/>
   <addaction name="actOpen"/>
   <addaction name="actSave"/>
   <addaction name="actExport"/>
   <addaction name="separator"/>
   <addaction name="actLoadImage"/>
   <addaction name="actCloseImage"/>
   <addaction name="separator"/>
   <addaction name="actResetView"/>
   <addaction name="separator"/>
   <addaction name="actModeView"/>
   <addaction name="actModeEdit"/>
   <addaction name="actModeDefineCentroid"/>
   <addaction name="separator"/>
   <addaction name="actSettings"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Objets</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeView" name="objectsView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actLoadImage">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/icons/open</normaloff>:/icons/open</iconset>
   </property>
   <property name="text">
    <string>Load &amp;Image</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actExit">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/icons/exit</normaloff>:/icons/exit</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actModeView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/icons/view</normaloff>:/icons/view</iconset>
   </property>
   <property name="text">
    <string>&amp;View mode</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actModeEdit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/icons/edit</normaloff>:/icons/edit</iconset>
   </property>
   <property name="text">
    <string>&amp;Edition mode</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actCloseImage">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/icons/close</normaloff>:/icons/close</iconset>
   </property>
   <property name="text">
    <string>&amp;Close Image</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actResetView">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/icons/reset_view</normaloff>:/icons/reset_view</iconset>
   </property>
   <property name="text">
    <string>&amp;Default View</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actOpen">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/icons/open</normaloff>:/icons/open</iconset>
   </property>
   <property name="text">
    <string>&amp;Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actSave">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/icons/save</normaloff>:/icons/save</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actNew">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/icons/new</normaloff>:/icons/new</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actSaveAs">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/icons/save</normaloff>:/icons/save</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actExport">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/icons/export</normaloff>:/icons/export</iconset>
   </property>
   <property name="text">
    <string>&amp;Export</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actSettings">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/icons/settings</normaloff>:/icons/settings</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actModeDefineCentroid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/icons/mesure</normaloff>:/icons/mesure</iconset>
   </property>
   <property name="text">
    <string>Define &amp;Centro√Ød mode</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actExportOption">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/icons/export</normaloff>:/icons/export</iconset>
   </property>
   <property name="text">
    <string>Export (advanced)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QWidget</extends>
   <header>imageView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/icons.qrc"/>
 </resources>
 <connections/>
</ui>
